tests/jenkins/CommonPipelineTest.groovy

```diff
@@ -28,4 +33,18 @@ abstract class CommonPipelineTest extends DeclarativePipelineTest {
         assertJobStatusSuccess()
         printCallStack()
     }
+
+    /**
+     * The author of signArtifacts is responsible for this method.
+     */
+    void verifySignArtifacts() {
```

Comment:
I like these asserts, we should definitely keep these. These act as a good enforcer.

Although, I still think we should not mock our code for the same reasons as before. 

https://github.com/opensearch-project/opensearch-build/pull/1471#discussion_r785093557